### PR TITLE
Header Component

### PR DESCRIPTION
**Pull request by Dulaya**
Created Header component.

### CODE REVIEW

**Comment by Dulaya** [comment]
Forgot to separate styles file from component file. Should be easy fix.

**Review by Nap1400 — APPROVED** [review]
All clear.

**Review by raynesmike — APPROVED** [review]
I approve with the header addition

File: src/components/header/Header.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    return(
+        <View style={style.container} {...rest} >
+            <View>
+                {/*Conditional rendering leftIcon*/}
+                {leftIcon && leftIcon}
+            </View>
+            <View>
+                {/*Check if title exists. If so, check type and render conditionally.*/}
+                {title && (typeof title === 'string'? <Text>{title}</Text>:title )}
```
Comment (carizzuti) [inline src/components/header/Header.js:15]: Your use of the conditional rendering helped me understand the concept better

**Review by carizzuti — APPROVED** [review]

**Comment by Dulaya** [comment]
Screenshot_1647558873

**Review by leo-he-1 — APPROVED** [review]
Looks good

**Pull request closed without merging**